Tasks/XamarinTestCloud/task.loc.json

```diff
@@ -105,8 +105,10 @@
       "required": true,
       "groupName": "advanced",
       "options": {
-        "en_US": "English (United States)",
```

Comment:
Originally this option is self explanatory; now with the "user defined" option, should we add a help markdown element?  Xamarin test cloud's UI states: If your language isn't displayed, please use any two-letter ISO-639 language code along with any two-letter ISO 3166 country code in the format [language]_[country].

We can add something similar?